packages/main/src/InputPopover.hbs

```diff
@@ -2,6 +2,7 @@
 	<ui5-responsive-popover
 		no-arrow
 		_disable-initial-focus
+		prevent-focus-restore
```

Comment:
This could be a problem. 

See Input.js#555
```js
	this.Suggestions.toggle(shouldOpenSuggestions, {
				preventFocusRestore: !this.hasSuggestionItemSelected,
			});
```

The preventFocusRestore is kind of optional and if set in the hbs, it will overwrite the code above, I should test if everything works as expected. This is related to QuickCard topic and opening an additional popover by suggestion item

Reply:
I removed it